src/rules/noObjectLiteralTypeAssertionRule.ts

```diff
@@ -37,23 +49,44 @@ export class Rule extends Lint.Rules.AbstractRule {
             The compiler will warn for excess properties with this syntax, but not missing required fields.
             For example: \`const x: { foo: number } = {}\` will fail to compile, but
             \`const x = {} as { foo: number }\` will succeed.`,
-        optionsDescription: "Not configurable.",
-        options: null,
-        optionExamples: [true],
+        optionsDescription: Lint.Utils.dedent`
+            One option may be configured:
+
+            * \`${OPTION_ALLOW_ARGUMENTS}\` allows type assertions to be used on object literals inside call expressions.`,
+        options: {
+            type: "array",
```

Comment:
you don't need to wrap the object in an array, you can just bring the object out as the top level `options` value

Reply:
Done! Thank you.